packages/syft/src/syft/core/node/worker.py

```diff
@@ -533,6 +538,32 @@ def queue_task(
     return None
 
 
+def create_worker_metadata(
+    worker: NewNode,
+) -> Optional[NodeMetadata]:
+    try:
+        metadata_stash = MetadataStash(store=worker.document_store)
+        metadata_exists = metadata_stash.get_all().ok()
+        if metadata_exists:
+            return None
+        else:
+            new_metadata = NodeMetadata(
+                name=worker.name,
+                id=worker.id,
+                verify_key=worker.signing_key.verify_key,
+                highest_object_version=HIGHEST_SYFT_OBJECT_VERSION,
+                lowest_object_version=LOWEST_SYFT_OBJECT_VERSION,
+                syft_version=__version__,
+                deployed_on=datetime.now().date().strftime("%m/%d/%Y"),
+            )
+            result = metadata_stash.set(metadata=new_metadata)
+            if result.is_ok():
+                return result.ok()
+            return None
+    except Exception as e:
+        print("create_worker_metadata failed", e)
+
+
```

Comment:
@madhavajay  I created this method to persist the metadata for the first time. I chose to declare it here since we also declared the `create_admin_new` method in this same location. Do you think we should move it to another location instead? Let me know your thoughts.